adam/adam_6224.py

```diff
@@ -0,0 +1,628 @@
+
+# -*- coding: utf-8 -*-
+#
+# This file is part of the dev-solaris-adam project
+#
+#
+#
+# Distributed under the terms of the LGPL license.
+# See LICENSE.txt for more info.
+
+
+__all__ = ["ADAM6224", "main"]
+
+# PyTango imports
+import tango
+from tango import DebugIt
+from tango.server import run
+from tango.server import Device, DeviceMeta
+from tango.server import attribute, command
+from tango.server import class_property, device_property
+from tango import AttrQuality, AttrWriteType, DispLevel, DevState
+# Additional import
+# PROTECTED REGION ID(ADAM6224.additional_import) ENABLED START #
+from pymodbus.client.sync import ModbusTcpClient
+from pymodbus.exceptions import ModbusException
+
+#mport time
+# PROTECTED REGION END #    //  ADAM6224.additional_import
+
+
+class ADAM6224(Device):
+    """ ADAM6224
+It is a definition of a class used to control ADAM-6224 controller via Modbus TCP.
+Device contains 4-ch Isolated Analog Output and 4-ch Digital Input.
+For each AO there are associated attributes:
+ - AnalogOutput(double) - contains present output value of the channel (V or mA), it is adjustable
```

Comment:
I'm fairly certain that this list will not be rendered correctly in Sphinx.
Please surround the lists in docstrings with empty lines and begin items with an asterisk, as in:
```
The line preceding a list.

* the first element,
* the second element

The line after a list.
```